pkg/compute/rust.go

```diff
@@ -218,10 +218,14 @@ func (r Rust) Verify(out io.Writer) error {
 	if err != nil {
 		return fmt.Errorf("error fetching latest crate version: %w", err)
 	}
-	latestFastlySys, err := getLatestCrateVersion(r.client, "fastly-sys")
+
+	latestFastlySys, err := semver.NewVersion(r.config.File.Language.Rust.FastlySys)
```

Comment:
I believe this should be `semver.NewConstraint` in order to parse the kinds of ranges we'll want to support. For example, we want to make sure that the fastly-sys in a package is `>= 0.3.0 && < 0.5.0`, not that it equals a precise version.

Reply:
Aha, ok gotcha. I'll fix that up 👍🏻

Reply:
Actually I just realize that we do create a constraint on a line just following this section. But it looks like we need an explicit min and max version for fastly-sys rather than constraining by a single value to the minor version.